lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -299,6 +299,7 @@ export class NativeInteractionClient extends BaseInteractionClient {
             correlationId: this.correlationId,
             instance_aware: instanceAware,
             token_type: request.authenticationScheme,
+            windowTitleSubstring: document.title,
```

Comment:
This is guaranteed to be unique, that okay?

Reply:
I assume you meant it's *not* guaranteed to be unique? Yes, we're aware of that scenario and we're ok with the parenting potentially being wrong in this case (we don't have a better solution at the moment). eSTS may include some unique characters in their title in the future to mitigate this if it becomes an issue.

Reply:
Yeah, sorry for the typo. Cool, resolving.